Resources/Prototypes/_RMC14/Roles/Jobs/Other/synth.yml

```
@@ -0,0 +1,87 @@
+- type: job
+  parent: CMJobBase
+  id: RMCJobSynthetic
+  name: rmc-job-name-synth
+  description: rmc-job-description-synth
+  playTimeTracker: RMCJobSynthetic
+  ranks:
+    RMCRankSyntheticMG:
+    - !type:RoleTimeRequirement
+      role: RMCJobSynthetic
+      time: 252000 # 70 hours
+    RMCRankSynthetic: []
+  startingGear: RMCGearSynthetic
+  icon: "RMCJobIconSynth"
+  requireAdminNotify: true
+  joinNotifyCrew: true
+  whitelisted: true
+  supervisors: cm-job-supervisors-co
+  accessGroups:
+  - ShipMasterAccess
+  minimapIcon:
+    sprite: _RMC14/Interface/map_blips.rsi
+    state: synth
+  minimapBackground:
+   sprite: _RMC14/Interface/map_blips.rsi
+   state: background_shipside
+  roleWeight: 0.25
+  special:
+  - !type:AddComponentSpecial
+    components:
+    - type: Skills
+      skills:
+        RMCSkillCqc: 5
+        RMCSkillEngineer: 4
+        RMCSkillConstruction: 3
+        # RMCSkillFirearms: 2 -- Synths cant use firearms why the fuck do they get the highest skill??
```

This is because the admins can toggle on their ability to use guns
```suggestion
        RMCSkillFirearms: 2
```